lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -1112,7 +1112,7 @@ def show_evasion_options(mod) # :nodoc:
           def show_plugins # :nodoc:
             tbl = Table.new(
               Table::Style::Default,
-              'Header'  => 'Plugins',
+              'Header'  => 'Loaded Plugins',
```

Comment:
Isn't this method supposed to show all the `available` plugins?

Reply:
In the pull request, yes. But this table generates loaded plugins, hence the title. I'll post a screenshot to make things clear.